--- TreasureBoxBP.json	2025-03-17 00:01:44.787111989 +0530
+++ -	2025-03-17 00:14:14.955000299 +0530
@@ -4752,7 +4752,7 @@
         },
         {
           "$type": "UAssetAPI.Kismet.Bytecode.Expressions.EX_JumpIfNot, UAssetAPI",
-          "CodeOffset": 153,
+          "CodeOffset": 81,
           "BooleanExpression": {
             "$type": "UAssetAPI.Kismet.Bytecode.Expressions.EX_LocalVariable, UAssetAPI",
             "Variable": {
